app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -307,6 +307,11 @@ class Settings private constructor(
         default = 0
     )
 
+    var totalUriCount by longPreference(
+        appContext.getPreferenceKey(R.string.pref_key_total_uri),
+        default = 0
```

Comment:
Is there a security risk here? We don't normally store telemetry data in prefs. 😬

Reply:
I don't think there's a security risk in this information here - number of urls opened is not PII or anything else.

Reply:
Do we ever reset this number? It's not clear to me that we do, and if we don't, this number will only be useful if we normalize it (e.g. 100 urls / days opened).